source/api_reference/version-history.html.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Version history
+weight: 0
+---
+# Version history
+
+## Version 1
+
+Version 1 can be accessed using either of these base URLs:
+
+- `https://{register-name}.register.gov.uk/v1/`
+- `https://{register-name}.register.gov.uk/`
+
+## Next version
+You can use this version of the API to test against backwards incompatable API changes before we release a new version of the API.
```

Comment:
Re. the term 'backwards incompatible', I assume users will know what such changes will be before they come to actually test this version of the API, so perhaps:

> You can use the 'Next' version of the API to test against backwards-incompatible changes before 'Next' becomes the stable API. 

I don't know if the 'Next' API will replace the older API entirely or whether the older API will still be hosted? The sentence I've recommended might sound slightly awkward but conventionally there's no "we/us" in the Registers docs at the moment.

Reply:
That wording is better, although I think it should say something like "before 'next' is renamed to the next stable API version" rather than just "becomes the stable API"

The timeline will be:
- we make changes to the 'next' version
- when we're happy with it, 'next' becomes version 2
- after a grace period, we get rid of version 1

Reply:
You could change the content up a bit and describe the timeline itself. For example:

> The release schedule is:
> 
> 1. The GOV.UK Registers team releases the 'Next' API in beta. You can use the 'Next' version of the API to test against backwards-incompatible changes before 'Next' becomes the stable API.
> 2. When the beta is concluded, the 'Next API' becomes the stable API (eg v2). 
> 3. After a set amount of time, the previous version of the API is deprecated.